lib/castle/commands/list/create.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Castle
+  module Commands
+    module List
+      class Create
+        class << self
+          # @param options [Hash]
+          # @return [Castle::Command]
+          def build(options = {})
+            Castle::Validators::Present.call(options, %i[name color primary_field])
+            # Castle::Validators::Allowed.call(options, %i[secondary_field description default_item_archivation_time])
```

Comment:
[note] this is just an idea of how we could expose the api expectation in code, but realize that it makes you have to rerelease the gem if the api changes

Reply:
Official API documentation would be a better source of valid params than a client code. So relying on the API response would be a better long-term option.